EMIP/1-1000/EMIP00816.xml

```diff
@@ -29,34 +29,226 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                         </extent>
+                        <collation>
+                        <list>
+                        <item xml:id="q1" n="A"><dim unit="leaf">1</dim></item>
+                        <item xml:id="q2" n="1"><dim unit="leaf">8</dim><locus from="1r" to="8v"></locus></item>
+                        <item xml:id="q3" n="2"><dim unit="leaf">8</dim><locus from="9r" to="16v"></locus></item>
+                        <item xml:id="q4" n="3"><dim unit="leaf">8</dim><locus from="17r" to="24v"></locus></item>
+                        <item xml:id="q5" n="4"><dim unit="leaf">8</dim><locus from="25r" to="32v"></locus></item>
+                        <item xml:id="q6" n="5"><dim unit="leaf">8</dim><locus from="33r" to="40v"></locus></item>
+                        <item xml:id="q7" n="6"><dim unit="leaf">8</dim><locus from="41r" to="48v"></locus></item>
+                        <item xml:id="q8" n="7"><dim unit="leaf">8</dim><locus from="49r" to="56v"></locus></item>
+                        <item xml:id="q9" n="8"><dim unit="leaf">8</dim><locus from="57r" to="64v"></locus></item>
+                        <item xml:id="q10" n="9"><dim unit="leaf">8</dim><locus from="65r" to="72v"></locus></item>
+                        <item xml:id="q11" n="10"><dim unit="leaf">8</dim><locus from="73r" to="80v"></locus></item>
+                        <item xml:id="q12" n="11"><dim unit="leaf">8</dim><locus from="81r" to="88v"></locus></item>
+                        <item xml:id="q13" n="12"><dim unit="leaf">8</dim><locus from="89r" to="96v"></locus></item>
+                        <item xml:id="q14" n="13"><dim unit="leaf">8</dim><locus from="97r" to="104v"></locus></item>
+                        <item xml:id="q15" n="14"><dim unit="leaf">8</dim><locus from="105r" to="112v"></locus></item>
+                        <item xml:id="q16" n="15"><dim unit="leaf">8</dim><locus from="113r" to="120v"></locus></item>
+                        <item xml:id="q17" n="16"><dim unit="leaf">8</dim><locus from="121r" to="128v"></locus></item>
+                        <item xml:id="q18" n="17"><dim unit="leaf">8</dim><locus from="129r" to="136v"></locus></item>
+                        </list>
+                        </collation>
+
+<condition key="good">
+<desc><locus target="#136r" facs="136">F. 136 is mutilated and damaged by humidity</locus>.</desc>
+</condition>
+
+
                      </supportDesc>
-                  </objectDesc>
+
+<layoutDesc>
+<layout columns="2" writtenLines="22"></layout>
+</layoutDesc>
+
+
+                   </objectDesc>
+<handDesc>
+<handNote xml:id="h1" script="Ethiopic">
+<seg type="ink">Black, red</seg>
+<seg type="rubrication">The names <foreign xml:lang="gez">ኢየሲስ፡</foreign> and <foreign xml:lang="gez">ማርያም፡</foreign>,
+titles of the homilies, elements of numerals and punctuation signs.</seg>
+</handNote>
+</handDesc>
+
+                  <additions>
+<list>
+
+
+<item xml:id="e1">
+<locus target="1r" facs="001"/>
```

Comment:
If you look in the guidelines, https://betamasaheft.eu/Guidelines/?id=additionsVaria, you will see that ownership notes should be coded as additions not as 'extra' or 'varia', and so should be given a id 'a1...'